src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -712,14 +712,15 @@
     <ILStrip
         Condition=" '$(WasmStripILAfterAOT)' == 'true' "
         TrimIndividualMethods="true"
+        IntermediateOutputPath="$(_WasmIntermediateOutputPath)"
         Assemblies="@(_WasmAssembliesInternal)">
-      <Output TaskParameter="TrimmedAssemblies" ItemName="_ILStripTrimmedAssemblies" />
+      <Output TaskParameter="UpdatedAssemblies" ItemName="_UpdatedAssembliesAfterILStrip" />
     </ILStrip>
-    <Move
-        Condition=" '$(WasmStripILAfterAOT)' == 'true' "
-        SourceFiles="@(_ILStripTrimmedAssemblies->'%(TrimmedAssemblyFileName)')"
-        DestinationFiles="@(_ILStripTrimmedAssemblies)"
-    />
+
+    <ItemGroup>
+      <_WasmAssembliesInternal Remove="@(_WasmAssembliesInternal)" Condition="'$(WasmStripILAfterAOT)' == 'true'" />
```

Comment:
nit: The condition can be moved to `ItemGroup`

Reply:
Updated.